packages/account/src/Constants/auth-status-codes.ts

```diff
@@ -1,6 +1,6 @@
-import { TVerificationStatus } from '../../../../Types';
+import { TVerificationStatus } from '../Types';
 
-export const poa_status_codes: TVerificationStatus = {
+export const AUTH_STATUS_CODES: TVerificationStatus = {
```

Comment:
i think its better we move this constant to shared package so that we can use it in other packages also.

Reply:
We should actually move to shared. Will update. Thanks @amina-deriv